classes/helper.php

```diff
@@ -79,4 +79,23 @@ public static function enable_feature_policy() {
             @header('Feature-Policy: ' . $featureheader);
         }
     }
+
+    public static function enable_notification_popup() : string {
+        global $PAGE, $USER;
+        $settings = get_config('local_csp');
+        if (empty($settings->popup_enable) || !has_capability('local/csp:seeviolationspopup', $PAGE->context, $USER->id)) {
+            return '';
+        }
+        $PAGE->requires->js_call_amd('local_csp/modal','init', [boolval($settings->popup_enforced_only)]);
+        return '
+        <script>
```

Comment:
Is there a good reason for this to be a hard coded inline script vs a normal amd script?

Reply:
Not saying there isn't, but if there is then please add a comment to the code for why it's done this way